### PR TITLE
Feature/atom upload

### PR DESCRIPTION
**Pull request by juanmaguitar**
Demo online can be checked at → https://sui-components-feature-atom-upload.now.sh/workbench/atom/upload/demo

### CODE REVIEW

File: components/atom/upload/src/index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+const capitalize = text => text[0].toUpperCase() + text.substr(1)
+
+class AtomUpload extends PureComponent {
+  state = {
+    Dropzone: null
+  }
+
+  loadAsyncReactDropzone() {
+    require.ensure(
```
Comment (carlosvillu) [inline components/atom/upload/src/index.js:25]: 👏👏

**Review by carlosvillu — DISMISSED** [review]

**Review by ddmarin94 — DISMISSED** [review]
🔝

File: components/atom/upload/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bg-atom-upload: #f5f5f5 !default;
+$c-atom-upload: #979797 !default;
+
+.sui-AtomUpload {
+  align-items: center;
+  background-color: $bg-atom-upload;
+  border: 2px dashed $c-atom-upload;
```
Comment (jcger) [inline components/atom/upload/src/index.scss:10]: please use sui-theme variables for this kind of stuff https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss#L5

File: components/atom/upload/src/index.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  color: $c-atom-upload;
+  display: flex;
+  flex-direction: column;
+  font-family: $ff-sans-serif;
```
Comment (jcger) [inline components/atom/upload/src/index.scss]: this should be set by the page or be a component variable

File: components/atom/upload/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  padding: $p-l * 3 $p-l * 2 $p-l;
+  text-align: center;
+
+  &-blockText {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h4,
```
Comment (jcger) [inline components/atom/upload/src/index.scss]: is this part of the wrapped component? Is there no class you could use as selector?

File: components/atom/upload/src/index.scss (hunk 2 above)
Comment (jcger) [reply components/atom/upload/src/index.scss:10]: as no borders are defined yet we'll do it this way but create a jira task in order to solve it as soon as they are defined

**Review by jcger — APPROVED** [review]

**Review by ddmarin94 — APPROVED** [review]

**Pull request merged**